src/modules/NtpClient.rb

```diff
@@ -1045,42 +1045,55 @@ def Summary
       summary
     end
 
-    # Test if specified NTP server answers
+
+    # Test if a specified NTP server is reachable by IPv4 or IPv6 (bsc#74076),
+    # Firewall could have been blocked IPv6
+    # @param [String] server string host name or IP address of the NTP server
+    # @return [Boolean] true if NTP server answers properly
+    def reachable_ntp_server?(server)
+      return true if sntp_test(server)
+
+      sntp_test(server, 6)
```

Comment:
Now (after redefining `sntp_test` and relocating the comments about the firewall) it looks more reasonable to reduce the body of this method to.
```ruby
sntp_test(server) || sntp_test(server, 6)
```